src/Features/LanguageServer/Protocol/Handler/Formatting/FormatDocumentOnTypeHandler.cs

```diff
@@ -68,10 +78,22 @@ public override async Task<TextEdit[]> HandleRequestAsync(DocumentOnTypeFormatti
             return edits.ToArrayAndFree();
         }
 
-        protected virtual Task<IList<TextChange>?> GetFormattingChangesOnReturnAsync(IEditorFormattingService formattingService, Document document, int position, CancellationToken cancellationToken)
+        protected virtual Task<IList<TextChange>?> GetFormattingChangesOnReturnAsync(
+            IEditorFormattingService formattingService,
+            Document document,
+            int position,
+            DocumentOptionSet documentOptions,
+            CancellationToken cancellationToken)
+            // We don't need to pass in options here since C#/VB always returns no text changes for this case.
```

Comment:
Will we need to at some point though? Or does F#/TypeScript need to? This is more a "if we're breaking the interface, do we need to break this too" sort of question.

Reply:
Hmm, good question. I guess it makes sense to add it just in case we do add support for this method in the future.